amd/src/management.js

```diff
@@ -396,8 +396,10 @@ const wipeModal = (event) => {
         try {
             await wipe();
             reload();
+            return;
```

Comment:
If you put return; here and in L402, return statement in L404 becomes unreachable and could be removed